posixlib/src/main/scala/scala/scalanative/posix/string.scala

```diff
@@ -1,8 +1,105 @@
 package scala.scalanative.posix
 
-import scalanative.unsafe.{extern, CInt, CString}
+import scala.scalanative.unsafe._
+import scala.scalanative.posix.sys.types
+
+// The Open Group Base Specifications Issue 7, 2018 edition
+
+/* A CX comment before a method indicates a POSIX extension to
+ * the ISO/IEEE C standard.
+ *
+ * An XSI comment before method indicates it is defined in
+ * extended POSIX X/Open System Interfaces, not base POSIX.
+ */
 
 @extern
 object string {
+  /* NULL is required by the POSIX standard but is not directly implemented
+   * here. It is implemented in posix/stddef.scala.
+   */
+
+  type size_t = types.size_t
+
+// CX - Begin
+  type locale_t = locale.locale_t
+// CX - End
```

Comment:
```suggestion
  /** CX */
  type locale_t = locale.locale_t
```

Reply:
Ah! I see what you are doing, I think.  You are making the comment as a scaladoc comment.

I can make those changes.  I still like having the "End" marker, especially for a block of CX or XSI
methods.  When does CX or XSI stop.  Especially relevant for the scaladoc info.

I could try to find the emojis for POSIX doc `[Option End]` symbol.